### PR TITLE
Revert "samples: nrf9160: Rename at_client -> at_host"

### PR DESCRIPTION
**Pull request by tejlmand**
This reverts commit 7a4e36676091951521b3aa7f9c2456de15971c31.

As discussed, renaming of samples frequently used is not a good
practice.
Therefore the at_client sample name will be preserved, and thus the
renaming commit is being reverted by this commit.

Signed-off-by: Torsten Rasmussen <Torsten.Rasmussen@nordicsemi.no>

### CODE REVIEW

File: samples/nrf9160/aws_fota/README.rst [hunk 1]
```diff
@@ -88,12 +88,12 @@ This application, which is part of `nRF Connect for Desktop`_, provides a certif
...
-#. Paste the three certificates into the respective fields.
-#. Choose a security tag.
-#. Click :guilabel:`Update certificates`.
-#. Before programming the sample, make sure to configure the :option:`security tag <CONFIG_CERT_SEC_TAG>` to the one that you chose.
+   1. Make sure that you have the AT client sample programmed on your kit.
+   #. Put the modem in offline state.
+   #. Paste the three certificates into the respective fields.
+   #. Choose a security tag.
+   #. Click :guilabel:`Update certificates`.
+   #. Before programming the sample, make sure to configure the :option:`security tag <CLOUD_CERT_SEC_TAG>` to the one that you chose.
```
Comment (jhn-nordic) [inline samples/nrf9160/aws_fota/README.rst:96]: should this be changed?

**Review by shantha-14 — APPROVED** [review]

File: samples/nrf9160/aws_fota/README.rst (hunk 1 above)
Comment (tejlmand) [reply samples/nrf9160/aws_fota/README.rst:96]: could very well be.
There were some minor conflicts on the release rst, and this rst file, so I chose what appeared to be the best option.

But I have no idea what should gi here instead. @umapraseeda or @bjda suggestions ?

**Review by jhn-nordic — APPROVED** [review]

**Review by Rallare — APPROVED** [review]

**Review by OeySa — APPROVED** [review]
LGTM

**Review by umapraseeda — APPROVED** [review]

File: samples/nrf9160/at_client/src/main.c [hunk 2]
```diff
@@ -31,5 +31,5 @@ void enable_xtal(void)
 void main(void)
 {
 	enable_xtal();
-	printk("AT host sample started\n");
+	printk("The AT host sample started\n");
```
Comment (didrikr) [inline samples/nrf9160/at_client/src/main.c:34]: Should this be changed to "AT client" as well?

Comment (tejlmand) [reply samples/nrf9160/at_client/src/main.c:34]: This was how it looked originally, so I would say changing this is outside the scope of a revert commit.

Comment (tejlmand) [reply samples/nrf9160/at_client/src/main.c:34]: FYI:
This is how it looked even before `at_client` --> `at_host` rename.
So it has always printed `AT host` at this location.
https://github.com/nrfconnect/sdk-nrf/blob/6956ad1850fd433c28f87d09629fc138f945a965/samples/nrf9160/at_client/src/main.c#L34

**Review by didrikr — APPROVED** [review]
LGTM

**Review by carlescufi — APPROVED** [review]

**Pull request merged**